.circleci/common.sh

```diff
@@ -111,10 +114,19 @@ function run_torch_xla_tests() {
 
     # GPU tests
     if [ -x "$(command -v nvidia-smi)" ]; then
-      echo "Running Syncfree Optimizer Test"
-      python test/test_syncfree_optimizers.py
-      # echo "Running MNIST Test"
-      # python test/test_train_mp_mnist_amp.py --fake_data
+      # Syncfree SGD optimizer tests
+      if [ -d ./amp/torch_xla/amp/syncfree]; then
+        echo "Running Syncfree Optimizer Test"
+        python test/test_syncfree_optimizers.py
+
+        # Following test scripts are mainly useful for
+        # performance evaluation & comparison among different
+        # amp optimizers.
+        # echo "Running MNIST Test"
+        # python test/test_train_mp_mnist_amp.py --fake_data
+        # echo "Running ImageNet Test"
+        # python etst/test_train_mp_imagenet_amp.py --fake_data
```

Comment:
mnist is disabled for a different reason, we can run `imagenet_amp` when it is gpu setup

Reply:
Make sure you only run 1 epoch through, otherwise it might take too long

Reply:
Created #3233